src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,75 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
```

Comment:
The `@OneToMany` annotation for the `tickets` field should include a `mappedBy` attribute to specify the field that owns the relationship in the `Ticket` entity. Additionally, consider specifying a fetch type, such as `FetchType.LAZY`, to optimize performance.